CorsixTH/Lua/window.lua

```diff
@@ -973,9 +973,11 @@ function Textbox:setActive(active)
     self.cursor_state = true
     self.cursor_pos[1] = type(self.text) == "table" and #self.text or 1
     self.cursor_pos[2] = type(self.text) == "table" and string.len(self.text[#self.text]) or string.len(self.text)
+    require("sdl").startTextInput()
```

Comment:
Any particular reason for this inline import? I'd prefer doing imports at the top of the file to avoid endless searches for imports

Reply:
That's not how lua works, require caches the object in packages.loaded so every subsequent call pulls up the same object - it can even store state.

I picked up the convention from https://github.com/pkulchenko/MobDebug, but I concede we don't use it elsewhere and it isn't entirely clear to people who aren't read up on lua - I'll change it if the functions work.